adoc/chapters/programming_interface.adoc

```diff
@@ -6507,6 +6517,15 @@ the range starts at the beginning of the [code]#buffer#.  The optional
 Throws an [code]#exception# with the [code]#errc::invalid# error code if
 [code]#accessRange# exceeds the range of [code]#bufferRef# in any dimension.
 
+If the range is zero sized, this constructs an empty accessor which fulfills the
+following post-conditions:
+--
+  * [code]#(empty() == true)#
+  * All size queries return [code]#0#.
+  * The only iterator that can be obtained is [code]#nullptr#.
+  * Trying to access the underlying memory is undefined behavior.
+--
```

Comment:
Do we need similar wording for all the **non**-ranged accessors for the case when the buffer size is `0`?  I presume all these statements are true for that case also, right?

Alternatively, maybe we don't need to say this at all.  I think the behavior is clear even without these statements.

Reply:
If the intent is clear, then I don't think we need to repeat this many, many time in the spec. I've removed this in 8fd20e133060b99cda654c45924b2253fe615036